projects/1.18/assets/farmers-respite/farmersrespite/lang/zh_cn.json

```
@@ -0,0 +1,72 @@
+{
+  "block.farmersrespite.kettle": "水壶",
+  "block.farmersrespite.rose_hip_pie": "玫瑰果派",
+  "block.farmersrespite.coffee_cake": "咖啡蛋糕",
+  "block.farmersrespite.small_tea_bush": "茶籽",
+  "block.farmersrespite.coffee_bush": "咖啡豆",
+  "block.farmersrespite.wild_tea_bush": "野生茶树",
+
+  "item.farmersrespite.green_tea_leaves": "绿茶叶",
+  "item.farmersrespite.yellow_tea_leaves": "黄茶叶",
+  "item.farmersrespite.black_tea_leaves": "红茶叶",
+  "item.farmersrespite.rose_hips": "玫瑰果",
+  "item.farmersrespite.tea_seeds": "茶籽",
+  "item.farmersrespite.green_tea": "绿茶",
+  "item.farmersrespite.yellow_tea": "黄茶",
+  "item.farmersrespite.black_tea": "红茶",
+  "item.farmersrespite.rose_hip_tea": "玫瑰果茶",
+  "item.farmersrespite.dandelion_tea": "蒲公英茶",
+  "item.farmersrespite.purulent_tea": "脓眼茶",
+  "item.farmersrespite.rose_hip_pie_slice": "玫瑰果派",
+  "item.farmersrespite.green_tea_cookie": "绿茶饼干",
+  "item.farmersrespite.nether_wart_sourdough": "下界疣发酵面包",
+  "item.farmersrespite.black_cod": "红茶炖鳕鱼",
+  "item.farmersrespite.tea_curry": "茶咖喱",
+  "item.farmersrespite.coffee_berries": "咖啡浆果",
+  "item.farmersrespite.coffee": "咖啡",
+  "item.farmersrespite.coffee_cake_slice": "咖啡蛋糕",
```

```suggestion
  "item.farmersrespite.coffee_cake_slice": "咖啡蛋糕切片",
```
https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/db11504a0bab4d7260f584437adecd8f569e2d58/projects/1.18/assets/farmers-delight/farmersdelight/lang/zh_cn.json#L102
  "item.farmersrespite.coffee_cake_slice": "Slice of Coffee Cake",